### DISCUSSION TITLE
Need a support for Mac M1

### DISCUSSION THREAD

**Discussion by abhishek-mittal** [markshust/docker-magento#415] [Ideas]
Facing problem while working on M1

currently using platform settings.

**Comment by markshust** [comment]
M1 support doesn't have anything to do with this project, and there's nothing I can do to help with it. You'll need to watch for official Docker for Mac support. This may be helpful https://docs.docker.com/docker-for-mac/apple-m1/

**Comment by markshust** [comment]
@abhishek-mittal update on this, `40.0.0` supports M1 chips 😎

**Reply by lewisvoncken** [reply]
@markshust 
can you share the required changes?

**Reply by markshust** [reply]
There are brand new images/builds, so you'll need to update your Docker images for them to take effect. You can do this by running `bin/update`, but note that any changes made to your Docker compose/bin folders will be updated.

Or, you can see some of the related PRs at https://github.com/markshust/docker-magento/pulls?q=is%3Apr+sort%3Aupdated-desc+arm+is%3Aclosed

**Comment by dcbaldwin1** [comment]
I know I'm late to the party but the current version of Mark's docker magento works fine for me on a Mac Studio with an M1 chip. Running on Docker Desktop 4.19 too.
